x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/update.ts

```diff
@@ -15,17 +15,17 @@ type RuleUpdatesBody = Pick<
   RuleUpdates,
   'name' | 'tags' | 'schedule' | 'actions' | 'params' | 'throttle' | 'notifyWhen'
 >;
```

Comment:
i think we can remove the `throttle` and `notifyWhen` fields from UI and BE contracts and set them as null in the rulesClient. WDYT?

Reply:
These properties are only deprecated for now, we're not ready to break the user calling these APIs with legacy props